### DISCUSSION TITLE
clone主题在哪儿？

### DISCUSSION THREAD

**Discussion by zhl111** [hugo-next/hugo-theme-next#149] [Q&A]
1.Vercel我部署完了
2.clone主题我找不到，这个主题如何与我部署的网站相关联？
3.git clone在哪里执行？用gitcmd吗？git clone后面需要添加我部署的网址吗？我在pc上建立一个文件夹为什么就不能clone呢？说需要.git

**Comment by elkan1788** [comment]
参考给你邮件回答信息和主题的REDME文件内容，能够正常的使用本主题。另外关于git的操作属于代码管理的技术基础，可以自行网络上搜索下相关资料进行学习。
